addons/sale_timesheet/models/sale_order_line.py

```diff
@@ -115,13 +112,12 @@ def _timesheet_create_project(self):
         uom_hour = self.env.ref('uom.product_uom_hour')
 
         # dict of inverse factors for each relevant UoM found in SO
-        factor_inv_per_id = {
-            uom.id: uom.factor_inv
+        factor_per_id = {
```

Comment:
Not sure that dict still has use once we remove the check on the `product_uom` category. It could filter out lines, but nw `line.product_uom_id.id in factor_per_id` will always be true anyway.
Or we're missing some kind of check?